server/models/conference.js

```diff
@@ -8,12 +8,15 @@ let conferenceSchema = new mongoose.Schema({
   defaultConf: Boolean,
   title: String,
   dateRange: {
+    _id: false,
     start: String, // 2016-12-30
     end: String
   },
   days: [{
+    _id: false,
     date: String,
     timeSlots: [{
+      _id: false,

```

Comment:
I rely on this ID extensively in my identification of timeslots, and this will break a lot of things. See: admin.service.ts lines 169-187

It would require an extensive refactor to remove dependence on this _id that I don't think we should be committing to at this point in the project.